stories/message-box/__snapshots__/message-box.stories.storyshot

```diff
@@ -137,7 +137,7 @@ exports[`Storyshots Components/Message Box No icon 1`] = `
             
                     
             <h2
-              class="fd-title fd-title--h5"
+              class="fd-title fd-title--h5 fd-message__title"
```

Comment:
I think you haven't updated the storyshot as this should be ` fd-message-box__title`, not `fd-message__title`